UPGRADE.md

```diff
@@ -16,13 +18,14 @@ Please note that users need to login again after this update, as it'll create a
 
 We now offer a Docker container that can be used to run Control Center, this is the recommended hosting now that ensures correct environement. If you still prefer to run it without Docker, this is still possible by cloning the repo and building the project manually.
 
-*Note: The included docker-compose.yaml is only for development purposes and should not be used in production.*
+In the instructions where we use `docker exec` we assume your container is named `control-center`. If you have named differently, please replace this.
```

Comment:
hmm, wasn't this written in another file too? 👀

Reply:
I think we could keep the Docker setup instructions outside of the upgrade, as it's not actually part of the upgrade: you don't _need_ to switch to it, and you can do it whenever you feel like it.

Reply:
Done